lib/linux_admin/logical_volume.rb

```diff
@@ -54,7 +54,6 @@ def extend_with(vg)
       self
     end
 
-    private
```

Comment:
The intention here is that the bytes_to_string method was made private.  Can you add `private_class_method` to it?

Reply:
Done.